poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java

```diff
@@ -751,7 +753,7 @@ private void syncRealtimeData(FacilitySoftwareConfig powerIQ) {
             return;
          }
       }
-      Map<String, Pdu> matchedPdus = new HashMap<String, Pdu>();
+      Map<String, String> pduAssetIdAndPduIdMap = new HashMap<String, String>();
```

Comment:
Why do we make those change?  since the pdu already contain the outlet informaiton.